### DISCUSSION TITLE
nuxi: command not found Error after rc.1 update

### DISCUSSION THREAD

**Discussion by antlionguard** [nuxt/bridge#923] [Q&A] [answered]
I got error after rc.1 update.

```
yarn run v1.22.19
$ nuxi dev
/bin/sh: nuxi: command not found
error Command failed with exit code 127.
```

@danielroe should i install `nuxi` into dev deps. or this error related with bridge?

**Comment by antlionguard** [comment]
And i got ` ERROR  Cannot start nuxt:  Cannot read properties of undefined (reading 'hook')` error if i install nuxi 3.9.0 version into devDeps. :/

**Reply by danielroe** [reply, marked as answer]
Yes, you should install Nuxt 3.8 explicitly. After next Nuxt release them latest nuxi will work; a polyfill has already landed upstream in kit.
